AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb

```diff
@@ -436,7 +415,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%load exercises/1.1-subplots_and_basic_plotting.py"
+    "# %load exercises/1.1-subplots_and_basic_plotting.py\n",
```

Comment:
We should keep these un-run in the canonical examples so we don't show the answers to the students too soon#